doc/storage.md

```diff
@@ -258,6 +237,31 @@ lxc storage create pool2 dir source=/data/lxd
   storage entities is not recommended by Ceph upstream. You may see unexpected
   and erratic failures which are unrelated to LXD itself.
 
+#### Storage pool configuration
+Key                             | Type      | Condition                         | Default                    | Description
+:--                             | :---      | :--------                         | :------                    | :----------
+volatile.pool.pristine          | string    | -                                 | true                       | Whether the pool has been empty on creation time.
```

Comment:
The config keys should be sorted alphabetically so this one should go to the end of the list, same for any other list of keys which isn't currently in alphabetical order.